layouts/shortcodes/implementation-status.html

```diff
@@ -0,0 +1,136 @@
+{{- /*
+  Shortcode to render implementation status tables from structured data.
+  Usage: {{< implementation-status >}}
+
+*/ -}}
```

Comment:
I also suggest here a breadcrumb to help people find the data without having to look at the source code. Something like

```suggestion
{{- /*
  Render implementation status tables from structured data in `data/implementation`
  Usage: {{< implementation-status >}}
*/ -}}
```

Reply:
done.